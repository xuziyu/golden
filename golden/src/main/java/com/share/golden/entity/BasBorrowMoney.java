package com.share.golden.entity;

public class BasBorrowMoney {
    private String borrowId;

    private String borrowSerial;

    private String userId;

    private String borrowCompany;

    private String borrowMoney;

    private String borrowRate;

    private String timeLimit;

    private String borrowLimit;

    private String borrowState;

    private String borrowRecommend;

    private String borrowType;

    private String borrowRemark1;

    private String borrowRemark2;

    public String getBorrowId() {
        return borrowId;
    }

    public void setBorrowId(String borrowId) {
        this.borrowId = borrowId == null ? null : borrowId.trim();
    }

    public String getBorrowSerial() {
        return borrowSerial;
    }

    public void setBorrowSerial(String borrowSerial) {
        this.borrowSerial = borrowSerial == null ? null : borrowSerial.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getBorrowCompany() {
        return borrowCompany;
    }

    public void setBorrowCompany(String borrowCompany) {
        this.borrowCompany = borrowCompany == null ? null : borrowCompany.trim();
    }

    public String getBorrowMoney() {
        return borrowMoney;
    }

    public void setBorrowMoney(String borrowMoney) {
        this.borrowMoney = borrowMoney == null ? null : borrowMoney.trim();
    }

    public String getBorrowRate() {
        return borrowRate;
    }

    public void setBorrowRate(String borrowRate) {
        this.borrowRate = borrowRate == null ? null : borrowRate.trim();
    }

    public String getTimeLimit() {
        return timeLimit;
    }

    public void setTimeLimit(String timeLimit) {
        this.timeLimit = timeLimit == null ? null : timeLimit.trim();
    }

    public String getBorrowLimit() {
        return borrowLimit;
    }

    public void setBorrowLimit(String borrowLimit) {
        this.borrowLimit = borrowLimit == null ? null : borrowLimit.trim();
    }

    public String getBorrowState() {
        return borrowState;
    }

    public void setBorrowState(String borrowState) {
        this.borrowState = borrowState == null ? null : borrowState.trim();
    }

    public String getBorrowRecommend() {
        return borrowRecommend;
    }

    public void setBorrowRecommend(String borrowRecommend) {
        this.borrowRecommend = borrowRecommend == null ? null : borrowRecommend.trim();
    }

    public String getBorrowType() {
        return borrowType;
    }

    public void setBorrowType(String borrowType) {
        this.borrowType = borrowType == null ? null : borrowType.trim();
    }

    public String getBorrowRemark1() {
        return borrowRemark1;
    }

    public void setBorrowRemark1(String borrowRemark1) {
        this.borrowRemark1 = borrowRemark1 == null ? null : borrowRemark1.trim();
    }

    public String getBorrowRemark2() {
        return borrowRemark2;
    }

    public void setBorrowRemark2(String borrowRemark2) {
        this.borrowRemark2 = borrowRemark2 == null ? null : borrowRemark2.trim();
    }
}