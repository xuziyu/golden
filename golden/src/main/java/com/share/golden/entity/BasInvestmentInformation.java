package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasInvestmentInformation {
    private String investorId;

    private String userId;

    private String tenderId;

    private BigDecimal investorMoney;

    private String investorStatus;

    private String investorStyle;

    private String brrowStatus;

    private String investorRest;

    private String profitModel;

    private BigDecimal profitMoney;

    private Date investorDate;

    private String replayDate;

    private String investorType;

    public String getInvestorId() {
        return investorId;
    }

    public void setInvestorId(String investorId) {
        this.investorId = investorId == null ? null : investorId.trim();
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId == null ? null : userId.trim();
    }

    public String getTenderId() {
        return tenderId;
    }

    public void setTenderId(String tenderId) {
        this.tenderId = tenderId == null ? null : tenderId.trim();
    }

    public BigDecimal getInvestorMoney() {
        return investorMoney;
    }

    public void setInvestorMoney(BigDecimal investorMoney) {
        this.investorMoney = investorMoney;
    }

    public String getInvestorStatus() {
        return investorStatus;
    }

    public void setInvestorStatus(String investorStatus) {
        this.investorStatus = investorStatus == null ? null : investorStatus.trim();
    }

    public String getInvestorStyle() {
        return investorStyle;
    }

    public void setInvestorStyle(String investorStyle) {
        this.investorStyle = investorStyle == null ? null : investorStyle.trim();
    }

    public String getBrrowStatus() {
        return brrowStatus;
    }

    public void setBrrowStatus(String brrowStatus) {
        this.brrowStatus = brrowStatus == null ? null : brrowStatus.trim();
    }

    public String getInvestorRest() {
        return investorRest;
    }

    public void setInvestorRest(String investorRest) {
        this.investorRest = investorRest == null ? null : investorRest.trim();
    }

    public String getProfitModel() {
        return profitModel;
    }

    public void setProfitModel(String profitModel) {
        this.profitModel = profitModel == null ? null : profitModel.trim();
    }

    public BigDecimal getProfitMoney() {
        return profitMoney;
    }

    public void setProfitMoney(BigDecimal profitMoney) {
        this.profitMoney = profitMoney;
    }

    public Date getInvestorDate() {
        return investorDate;
    }

    public void setInvestorDate(Date investorDate) {
        this.investorDate = investorDate;
    }

    public String getReplayDate() {
        return replayDate;
    }

    public void setReplayDate(String replayDate) {
        this.replayDate = replayDate == null ? null : replayDate.trim();
    }

    public String getInvestorType() {
        return investorType;
    }

    public void setInvestorType(String investorType) {
        this.investorType = investorType == null ? null : investorType.trim();
    }
}