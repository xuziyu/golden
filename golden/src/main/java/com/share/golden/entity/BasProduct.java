package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasProduct {
    private String productId;

    private BigDecimal productMoney;

    private String productIncome;

    private Date productTime;

    private String productType;

    private String productWay;

    private Date pcount;

    private String productProgress;

    private String productSaveway;

    private String productRateben;

    private Date productPublishTime;

    private String productName;

    private BigDecimal productTotalMoney;

    private String investmentScope;

    private String useOfProceeds;

    private String productState;

    private String productPicture;

    private String produceId;

    private String productDesc;

    private String productSafe;

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId == null ? null : productId.trim();
    }

    public BigDecimal getProductMoney() {
        return productMoney;
    }

    public void setProductMoney(BigDecimal productMoney) {
        this.productMoney = productMoney;
    }

    public String getProductIncome() {
        return productIncome;
    }

    public void setProductIncome(String productIncome) {
        this.productIncome = productIncome == null ? null : productIncome.trim();
    }

    public Date getProductTime() {
        return productTime;
    }

    public void setProductTime(Date productTime) {
        this.productTime = productTime;
    }

    public String getProductType() {
        return productType;
    }

    public void setProductType(String productType) {
        this.productType = productType == null ? null : productType.trim();
    }

    public String getProductWay() {
        return productWay;
    }

    public void setProductWay(String productWay) {
        this.productWay = productWay == null ? null : productWay.trim();
    }

    public Date getPcount() {
        return pcount;
    }

    public void setPcount(Date pcount) {
        this.pcount = pcount;
    }

    public String getProductProgress() {
        return productProgress;
    }

    public void setProductProgress(String productProgress) {
        this.productProgress = productProgress == null ? null : productProgress.trim();
    }

    public String getProductSaveway() {
        return productSaveway;
    }

    public void setProductSaveway(String productSaveway) {
        this.productSaveway = productSaveway == null ? null : productSaveway.trim();
    }

    public String getProductRateben() {
        return productRateben;
    }

    public void setProductRateben(String productRateben) {
        this.productRateben = productRateben == null ? null : productRateben.trim();
    }

    public Date getProductPublishTime() {
        return productPublishTime;
    }

    public void setProductPublishTime(Date productPublishTime) {
        this.productPublishTime = productPublishTime;
    }

    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName == null ? null : productName.trim();
    }

    public BigDecimal getProductTotalMoney() {
        return productTotalMoney;
    }

    public void setProductTotalMoney(BigDecimal productTotalMoney) {
        this.productTotalMoney = productTotalMoney;
    }

    public String getInvestmentScope() {
        return investmentScope;
    }

    public void setInvestmentScope(String investmentScope) {
        this.investmentScope = investmentScope == null ? null : investmentScope.trim();
    }

    public String getUseOfProceeds() {
        return useOfProceeds;
    }

    public void setUseOfProceeds(String useOfProceeds) {
        this.useOfProceeds = useOfProceeds == null ? null : useOfProceeds.trim();
    }

    public String getProductState() {
        return productState;
    }

    public void setProductState(String productState) {
        this.productState = productState == null ? null : productState.trim();
    }

    public String getProductPicture() {
        return productPicture;
    }

    public void setProductPicture(String productPicture) {
        this.productPicture = productPicture == null ? null : productPicture.trim();
    }

    public String getProduceId() {
        return produceId;
    }

    public void setProduceId(String produceId) {
        this.produceId = produceId == null ? null : produceId.trim();
    }

    public String getProductDesc() {
        return productDesc;
    }

    public void setProductDesc(String productDesc) {
        this.productDesc = productDesc == null ? null : productDesc.trim();
    }

    public String getProductSafe() {
        return productSafe;
    }

    public void setProductSafe(String productSafe) {
        this.productSafe = productSafe == null ? null : productSafe.trim();
    }
}