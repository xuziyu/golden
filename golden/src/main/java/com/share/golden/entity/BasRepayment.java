package com.share.golden.entity;

import java.util.Date;

public class BasRepayment {
    private String repayId;

    private String repayCode;

    private Date repayDate;

    private String repayState;

    private String repayTime;

    private String repayRemark1;

    private String repayRemark2;

    public String getRepayId() {
        return repayId;
    }

    public void setRepayId(String repayId) {
        this.repayId = repayId == null ? null : repayId.trim();
    }

    public String getRepayCode() {
        return repayCode;
    }

    public void setRepayCode(String repayCode) {
        this.repayCode = repayCode == null ? null : repayCode.trim();
    }

    public Date getRepayDate() {
        return repayDate;
    }

    public void setRepayDate(Date repayDate) {
        this.repayDate = repayDate;
    }

    public String getRepayState() {
        return repayState;
    }

    public void setRepayState(String repayState) {
        this.repayState = repayState == null ? null : repayState.trim();
    }

    public String getRepayTime() {
        return repayTime;
    }

    public void setRepayTime(String repayTime) {
        this.repayTime = repayTime == null ? null : repayTime.trim();
    }

    public String getRepayRemark1() {
        return repayRemark1;
    }

    public void setRepayRemark1(String repayRemark1) {
        this.repayRemark1 = repayRemark1 == null ? null : repayRemark1.trim();
    }

    public String getRepayRemark2() {
        return repayRemark2;
    }

    public void setRepayRemark2(String repayRemark2) {
        this.repayRemark2 = repayRemark2 == null ? null : repayRemark2.trim();
    }
}