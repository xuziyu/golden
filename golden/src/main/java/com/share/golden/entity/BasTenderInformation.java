package com.share.golden.entity;

import java.math.BigDecimal;
import java.util.Date;

public class BasTenderInformation {
	private String tenderId;

	private String userId;

	private String markcount;

	private String lmcount;

	private String gmcount;

	private BigDecimal markmoney;

	private BigDecimal markgain;

	private Date markbegindate;

	private Date markenddate;

	private Date date;

	public String getTenderId() {
		return tenderId;
	}

	public void setTenderId(String tenderId) {
		this.tenderId = tenderId == null ? null : tenderId.trim();
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId == null ? null : userId.trim();
	}

	public String getMarkcount() {
		return markcount;
	}

	public void setMarkcount(String markcount) {
		this.markcount = markcount == null ? null : markcount.trim();
	}

	public String getLmcount() {
		return lmcount;
	}

	public void setLmcount(String lmcount) {
		this.lmcount = lmcount == null ? null : lmcount.trim();
	}

	public String getGmcount() {
		return gmcount;
	}

	public void setGmcount(String gmcount) {
		this.gmcount = gmcount == null ? null : gmcount.trim();
	}

	public BigDecimal getMarkmoney() {
		return markmoney;
	}

	public void setMarkmoney(BigDecimal markmoney) {
		this.markmoney = markmoney;
	}

	public BigDecimal getMarkgain() {
		return markgain;
	}

	public void setMarkgain(BigDecimal markgain) {
		this.markgain = markgain;
	}

	public Date getMarkbegindate() {
		return markbegindate;
	}

	public void setMarkbegindate(Date markbegindate) {
		this.markbegindate = markbegindate;
	}

	public Date getMarkenddate() {
		return markenddate;
	}

	public void setMarkenddate(Date markenddate) {
		this.markenddate = markenddate;
	}

	public Date getDate() {
		return date;
	}

	public void setDate(Date date) {
		this.date = date;
	}
}