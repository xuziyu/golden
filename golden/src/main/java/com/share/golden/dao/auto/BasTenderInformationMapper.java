package com.share.golden.dao.auto;

import com.share.golden.entity.BasTenderInformation;
import com.share.golden.entity.BasTenderInformationExample;
import java.util.List;
import org.apache.ibatis.annotations.Param;

public interface BasTenderInformationMapper {
    long countByExample(BasTenderInformationExample example);

    int deleteByExample(BasTenderInformationExample example);

    int deleteByPrimaryKey(String tenderId);

    int insert(BasTenderInformation record);

    int insertSelective(BasTenderInformation record);

    List<BasTenderInformation> selectByExample(BasTenderInformationExample example);

    BasTenderInformation selectByPrimaryKey(String tenderId);

    int updateByExampleSelective(@Param("record") BasTenderInformation record, @Param("example") BasTenderInformationExample example);

    int updateByExample(@Param("record") BasTenderInformation record, @Param("example") BasTenderInformationExample example);

    int updateByPrimaryKeySelective(BasTenderInformation record);

    int updateByPrimaryKey(BasTenderInformation record);
}