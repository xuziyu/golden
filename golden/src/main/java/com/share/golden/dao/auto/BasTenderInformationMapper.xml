<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.share.golden.dao.auto.BasTenderInformationMapper">
  <resultMap id="BaseResultMap" type="com.share.golden.entity.BasTenderInformation">
    <id column="TENDER_ID" jdbcType="VARCHAR" property="tenderId" />
    <result column="USER_ID" jdbcType="VARCHAR" property="userId" />
    <result column="MARKCOUNT" jdbcType="VARCHAR" property="markcount" />
    <result column="LMCOUNT" jdbcType="VARCHAR" property="lmcount" />
    <result column="GMCOUNT" jdbcType="VARCHAR" property="gmcount" />
    <result column="MARKMONEY" jdbcType="DECIMAL" property="markmoney" />
    <result column="MARKGAIN" jdbcType="DECIMAL" property="markgain" />
    <result column="MARKBEGINDATE" jdbcType="TIMESTAMP" property="markbegindate" />
    <result column="MARKENDDATE" jdbcType="TIMESTAMP" property="markenddate" />
    <result column="DATE" jdbcType="TIMESTAMP" property="date" />
  </resultMap>
  <sql id="Example_Where_Clause">
    <where>
      <foreach collection="oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Update_By_Example_Where_Clause">
    <where>
      <foreach collection="example.oredCriteria" item="criteria" separator="or">
        <if test="criteria.valid">
          <trim prefix="(" prefixOverrides="and" suffix=")">
            <foreach collection="criteria.criteria" item="criterion">
              <choose>
                <when test="criterion.noValue">
                  and ${criterion.condition}
                </when>
                <when test="criterion.singleValue">
                  and ${criterion.condition} #{criterion.value}
                </when>
                <when test="criterion.betweenValue">
                  and ${criterion.condition} #{criterion.value} and #{criterion.secondValue}
                </when>
                <when test="criterion.listValue">
                  and ${criterion.condition}
                  <foreach close=")" collection="criterion.value" item="listItem" open="(" separator=",">
                    #{listItem}
                  </foreach>
                </when>
              </choose>
            </foreach>
          </trim>
        </if>
      </foreach>
    </where>
  </sql>
  <sql id="Base_Column_List">
    "TENDER_ID", "USER_ID", "MARKCOUNT", "LMCOUNT", "GMCOUNT", "MARKMONEY", "MARKGAIN", 
    "MARKBEGINDATE", "MARKENDDATE", "DATE"
  </sql>
  <select id="selectByExample" parameterType="com.share.golden.entity.BasTenderInformationExample" resultMap="BaseResultMap">
    select
    <if test="distinct">
      distinct
    </if>
    <include refid="Base_Column_List" />
    from BAS_TENDER_INFORMATION
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
    <if test="orderByClause != null">
      order by ${orderByClause}
    </if>
  </select>
  <select id="selectByPrimaryKey" parameterType="java.lang.String" resultMap="BaseResultMap">
    select 
    <include refid="Base_Column_List" />
    from BAS_TENDER_INFORMATION
    where "TENDER_ID" = #{tenderId,jdbcType=VARCHAR}
  </select>
  <delete id="deleteByPrimaryKey" parameterType="java.lang.String">
    delete from BAS_TENDER_INFORMATION
    where "TENDER_ID" = #{tenderId,jdbcType=VARCHAR}
  </delete>
  <delete id="deleteByExample" parameterType="com.share.golden.entity.BasTenderInformationExample">
    delete from BAS_TENDER_INFORMATION
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </delete>
  <insert id="insert" parameterType="com.share.golden.entity.BasTenderInformation">
    insert into BAS_TENDER_INFORMATION ("TENDER_ID", "USER_ID", "MARKCOUNT", 
      "LMCOUNT", "GMCOUNT", "MARKMONEY", 
      "MARKGAIN", "MARKBEGINDATE", "MARKENDDATE", 
      "DATE")
    values (#{tenderId,jdbcType=VARCHAR}, #{userId,jdbcType=VARCHAR}, #{markcount,jdbcType=VARCHAR}, 
      #{lmcount,jdbcType=VARCHAR}, #{gmcount,jdbcType=VARCHAR}, #{markmoney,jdbcType=DECIMAL}, 
      #{markgain,jdbcType=DECIMAL}, #{markbegindate,jdbcType=TIMESTAMP}, #{markenddate,jdbcType=TIMESTAMP}, 
      #{date,jdbcType=TIMESTAMP})
  </insert>
  <insert id="insertSelective" parameterType="com.share.golden.entity.BasTenderInformation">
    insert into BAS_TENDER_INFORMATION
    <trim prefix="(" suffix=")" suffixOverrides=",">
      <if test="tenderId != null">
        "TENDER_ID",
      </if>
      <if test="userId != null">
        "USER_ID",
      </if>
      <if test="markcount != null">
        "MARKCOUNT",
      </if>
      <if test="lmcount != null">
        "LMCOUNT",
      </if>
      <if test="gmcount != null">
        "GMCOUNT",
      </if>
      <if test="markmoney != null">
        "MARKMONEY",
      </if>
      <if test="markgain != null">
        "MARKGAIN",
      </if>
      <if test="markbegindate != null">
        "MARKBEGINDATE",
      </if>
      <if test="markenddate != null">
        "MARKENDDATE",
      </if>
      <if test="date != null">
        "DATE",
      </if>
    </trim>
    <trim prefix="values (" suffix=")" suffixOverrides=",">
      <if test="tenderId != null">
        #{tenderId,jdbcType=VARCHAR},
      </if>
      <if test="userId != null">
        #{userId,jdbcType=VARCHAR},
      </if>
      <if test="markcount != null">
        #{markcount,jdbcType=VARCHAR},
      </if>
      <if test="lmcount != null">
        #{lmcount,jdbcType=VARCHAR},
      </if>
      <if test="gmcount != null">
        #{gmcount,jdbcType=VARCHAR},
      </if>
      <if test="markmoney != null">
        #{markmoney,jdbcType=DECIMAL},
      </if>
      <if test="markgain != null">
        #{markgain,jdbcType=DECIMAL},
      </if>
      <if test="markbegindate != null">
        #{markbegindate,jdbcType=TIMESTAMP},
      </if>
      <if test="markenddate != null">
        #{markenddate,jdbcType=TIMESTAMP},
      </if>
      <if test="date != null">
        #{date,jdbcType=TIMESTAMP},
      </if>
    </trim>
  </insert>
  <select id="countByExample" parameterType="com.share.golden.entity.BasTenderInformationExample" resultType="java.lang.Long">
    select count(*) from BAS_TENDER_INFORMATION
    <if test="_parameter != null">
      <include refid="Example_Where_Clause" />
    </if>
  </select>
  <update id="updateByExampleSelective" parameterType="map">
    update BAS_TENDER_INFORMATION
    <set>
      <if test="record.tenderId != null">
        "TENDER_ID" = #{record.tenderId,jdbcType=VARCHAR},
      </if>
      <if test="record.userId != null">
        "USER_ID" = #{record.userId,jdbcType=VARCHAR},
      </if>
      <if test="record.markcount != null">
        "MARKCOUNT" = #{record.markcount,jdbcType=VARCHAR},
      </if>
      <if test="record.lmcount != null">
        "LMCOUNT" = #{record.lmcount,jdbcType=VARCHAR},
      </if>
      <if test="record.gmcount != null">
        "GMCOUNT" = #{record.gmcount,jdbcType=VARCHAR},
      </if>
      <if test="record.markmoney != null">
        "MARKMONEY" = #{record.markmoney,jdbcType=DECIMAL},
      </if>
      <if test="record.markgain != null">
        "MARKGAIN" = #{record.markgain,jdbcType=DECIMAL},
      </if>
      <if test="record.markbegindate != null">
        "MARKBEGINDATE" = #{record.markbegindate,jdbcType=TIMESTAMP},
      </if>
      <if test="record.markenddate != null">
        "MARKENDDATE" = #{record.markenddate,jdbcType=TIMESTAMP},
      </if>
      <if test="record.date != null">
        "DATE" = #{record.date,jdbcType=TIMESTAMP},
      </if>
    </set>
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByExample" parameterType="map">
    update BAS_TENDER_INFORMATION
    set "TENDER_ID" = #{record.tenderId,jdbcType=VARCHAR},
      "USER_ID" = #{record.userId,jdbcType=VARCHAR},
      "MARKCOUNT" = #{record.markcount,jdbcType=VARCHAR},
      "LMCOUNT" = #{record.lmcount,jdbcType=VARCHAR},
      "GMCOUNT" = #{record.gmcount,jdbcType=VARCHAR},
      "MARKMONEY" = #{record.markmoney,jdbcType=DECIMAL},
      "MARKGAIN" = #{record.markgain,jdbcType=DECIMAL},
      "MARKBEGINDATE" = #{record.markbegindate,jdbcType=TIMESTAMP},
      "MARKENDDATE" = #{record.markenddate,jdbcType=TIMESTAMP},
      "DATE" = #{record.date,jdbcType=TIMESTAMP}
    <if test="_parameter != null">
      <include refid="Update_By_Example_Where_Clause" />
    </if>
  </update>
  <update id="updateByPrimaryKeySelective" parameterType="com.share.golden.entity.BasTenderInformation">
    update BAS_TENDER_INFORMATION
    <set>
      <if test="userId != null">
        "USER_ID" = #{userId,jdbcType=VARCHAR},
      </if>
      <if test="markcount != null">
        "MARKCOUNT" = #{markcount,jdbcType=VARCHAR},
      </if>
      <if test="lmcount != null">
        "LMCOUNT" = #{lmcount,jdbcType=VARCHAR},
      </if>
      <if test="gmcount != null">
        "GMCOUNT" = #{gmcount,jdbcType=VARCHAR},
      </if>
      <if test="markmoney != null">
        "MARKMONEY" = #{markmoney,jdbcType=DECIMAL},
      </if>
      <if test="markgain != null">
        "MARKGAIN" = #{markgain,jdbcType=DECIMAL},
      </if>
      <if test="markbegindate != null">
        "MARKBEGINDATE" = #{markbegindate,jdbcType=TIMESTAMP},
      </if>
      <if test="markenddate != null">
        "MARKENDDATE" = #{markenddate,jdbcType=TIMESTAMP},
      </if>
      <if test="date != null">
        "DATE" = #{date,jdbcType=TIMESTAMP},
      </if>
    </set>
    where "TENDER_ID" = #{tenderId,jdbcType=VARCHAR}
  </update>
  <update id="updateByPrimaryKey" parameterType="com.share.golden.entity.BasTenderInformation">
    update BAS_TENDER_INFORMATION
    set "USER_ID" = #{userId,jdbcType=VARCHAR},
      "MARKCOUNT" = #{markcount,jdbcType=VARCHAR},
      "LMCOUNT" = #{lmcount,jdbcType=VARCHAR},
      "GMCOUNT" = #{gmcount,jdbcType=VARCHAR},
      "MARKMONEY" = #{markmoney,jdbcType=DECIMAL},
      "MARKGAIN" = #{markgain,jdbcType=DECIMAL},
      "MARKBEGINDATE" = #{markbegindate,jdbcType=TIMESTAMP},
      "MARKENDDATE" = #{markenddate,jdbcType=TIMESTAMP},
      "DATE" = #{date,jdbcType=TIMESTAMP}
    where "TENDER_ID" = #{tenderId,jdbcType=VARCHAR}
  </update>
</mapper>